eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java

```diff
@@ -112,15 +112,15 @@ public void testSubscribeWithUrl() {
     @Test
     public void testSubscribeStreamWithoutListener() {
         eventMeshGrpcConsumer.subscribe(Collections.singletonList(buildMockSubscriptionItem()));
-        verifyZeroInteractions(consumerAsyncClient);
+        verifyNoMoreInteractions(consumerAsyncClient);
```

Comment:
I figure it is better use verifyZeroInteractions method here.

Reply:
verifyZeroInteractions is a deprecated method that also calls verifyNoMoreInteractions at the bottom.
<img width="602" alt="image" src="https://user-images.githubusercontent.com/42484192/204566834-bfcb29f3-d8f3-4c16-9f65-593edac1fa19.png">

Reply:
ok